modules/nf-core/glimpse2/phase/main.nf

```diff
@@ -33,7 +33,7 @@ process GLIMPSE2_PHASE {
     def region = input_region    ? "${output_region.replace(":","_")}" : "${reference}"
     def args   = task.ext.args   ?: ""
     def prefix = task.ext.prefix ?: "${meta.id}_${region}"
-    def suffix = task.ext.suffix ?: "bcf"
+    def suffix = task.ext.suffix ?: "vcf.gz"
```

Comment:
why did this default change?

Reply:
It was to standardize with the other module.
Which should I prefer ?

Reply:
Umm, no idea ;) 
Maybe ask on maintainers team.

Reply:
I get consistency is good, but is someone using these modules/subwf in a pipeline?

Reply:
Yeah, me in phaseimpute 😆

Reply:
Hah, ok, then your call ;)

Reply:
I've sent a message on the slack channel 😉